server/src/lint.ts

```diff
@@ -16,9 +16,22 @@ export default async function lint(
     return JSON.parse(result)
   } catch (errors) {
     const errorMessage = "prisma-fmt error'd during linting.\n"
+
     if (onError) {
-      onError(errorMessage + errors)
+      if (
+        (errors as string) &&
+        errors.includes('ParsingError') &&
+        text.includes('type')
+      ) {
+        // recognized Prisma 1 schema
+        onError(
+          "You are currently viewing a Prisma 1 datamodel which is based on GraphQL syntax. The current Prisma VS Code extension doesn't support this syntax. To get proper syntax highlighting for this file, please change its file extension to `.graphql` and download the [GraphQL VS Code extension](https://marketplace.visualstudio.com/items?itemName=Prisma.vscode-graphql). Learn more [here](https://pris.ly/prisma1-vscode).",
```

Comment:
A few minor english things:

- `based on GraphQL` => `based on the GraphQL`
- `VS Code` => `VSCode`
- `change its file` => `change the file extension`